spec/author_spec.rb

```diff
@@ -0,0 +1,20 @@
+require_relative '../author'
+require_relative '../item'
+
+describe Author do
+  before :each do
+    @new_author = Author.new('flo', 'vitalis')
+  end
+
+  it 'should be an instance of Author' do
+    expect(@new_author).to be_an_instance_of(Author)
+  end
+
+  it 'should have a first name' do
+    expect(@new_author.first_name).to be_a(String)
+  end
+
+  it 'should have a last name' do
+    expect(@new_author.last_name).to be_a(String)
+  end
+end
```

Comment:
- [ ] Kindly make sure that all public methods in all classes have unit tests that cover all edge cases, as per [this](https://github.com/microverseinc/curriculum-ruby/blob/main/group-capstone/catalog_of_my_things.md#:~:text=Add%20unit%20tests%20for%20all%20implemented%20methods.) requirement.